make setup issue "ERROR: field <@timestamp> is duplicated, remove it or set 'overwrite: true'

Not sure if this is the correct place for this issue. I cloned the repo to go/src/github.com instead of go/github.com, since the src directory is where all the other packages were installed. Any help is appreciated. Thank you,

ubuntu % make setup
mkdir -p vendor/github.com/elastic/
cp -R /home/bwillcox/go/src/github.com/elastic/beats vendor/github.com/elastic/
rm -rf vendor/github.com/elastic/beats/.git
make update
make[1]: Entering directory '/home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat'
Running virtualenv with interpreter /usr/bin/python2
New python executable in /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/build/python-env/bin/python2
Also creating executable in /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/build/python-env/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
Generated fields.yml for prometheusbeat to /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/fields.yml
Updating generated files for prometheusbeat
mkdir -p include
go run  ./vendor/github.com/elastic/beats/dev-tools/cmd/asset/asset.go -license ASL2 -pkg include -in fields.yml -out include/fields.go prometheusbeat
if [ -d /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/_meta/kibana ]; then \
    cp -pr /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/_meta/kibana/* /home/bwillcox/go/src/github.com/infonova/prometheusbeat/pr\
ometheusbeat/_meta/kibana.generated ; \
fi
ERROR: field <@timestamp> is duplicated, remove it or set 'overwrite: true'
exit status 1
vendor/github.com/elastic/beats/libbeat/scripts/Makefile:315: recipe for target 'update' failed
make[1]: *** [update] Error 1
make[1]: Leaving directory '/home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat'
Makefile:18: recipe for target 'setup' failed
make: *** [setup] Error 2
ubuntu % go version
go version go1.10.4 linux/amd64
ubuntu % uname -a
Linux ubuntu 4.15.0-42-generic #45-Ubuntu SMP Thu Nov 15 19:32:57 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

Here's what I get...

ubuntu % pwd
/home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat
ubuntu % make
go build -i -ldflags "-X github.com/elastic/beats/libbeat/version.buildTime=2019-01-07T15:32:43Z -X github.com/elastic/beats/libbeat/version.commit=a65c77cf4\
4910e4b1385cb63549c8171495a7dae"
main.go:6:2: cannot find package "github.com/infonova/prometheusbeat/cmd" in any of:
    /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/vendor/github.com/infonova/prometheusbeat/cmd (vendor tree)
    /usr/lib/go-1.10/src/github.com/infonova/prometheusbeat/cmd (from $GOROOT)
    /home/bwillcox/go/src/github.com/infonova/prometheusbeat/cmd (from $GOPATH)
main.go:8:2: cannot find package "github.com/infonova/prometheusbeat/include" in any of:
    /home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat/vendor/github.com/infonova/prometheusbeat/include (vendor tree)
    /usr/lib/go-1.10/src/github.com/infonova/prometheusbeat/include (from $GOROOT)
    /home/bwillcox/go/src/github.com/infonova/prometheusbeat/include (from $GOPATH)
vendor/github.com/elastic/beats/libbeat/scripts/Makefile:100: recipe for target 'prometheusbeat' failed
make: *** [prometheusbeat] Error 1
ubuntu % ls
total 228
...
drwxrwxr-x 3 bwillcox bwillcox  4096 Dec 28 16:38 build
drwxrwxr-x 2 bwillcox bwillcox  4096 Dec 28 16:37 **cmd**
drwxrwxr-x 2 bwillcox bwillcox  4096 Dec 28 16:37 config
...
ubuntu % go get github.com/infonova/prometheusbeat/cmd
package github.com/infonova/prometheusbeat/cmd: /home/bwillcox/go/src/github.com/infonova/prometheusbeat exists but /home/bwillcox/go/src/github.com/infonova\
/prometheusbeat/.git does not - stale checkout?
ubuntu % ls .git
total 860
drwxrwxr-x 2 bwillcox bwillcox   4096 Dec 28 16:37 branches
-rw-rw-r-- 1 bwillcox bwillcox    267 Dec 28 16:37 config
-rw-rw-r-- 1 bwillcox bwillcox     73 Dec 28 16:37 description
-rw-rw-r-- 1 bwillcox bwillcox     23 Dec 28 16:37 HEAD
drwxrwxr-x 2 bwillcox bwillcox   4096 Dec 28 16:37 hooks
-rw-rw-r-- 1 bwillcox bwillcox 838375 Dec 28 17:14 index
drwxrwxr-x 2 bwillcox bwillcox   4096 Dec 28 16:37 info
drwxrwxr-x 3 bwillcox bwillcox   4096 Dec 28 16:37 logs
drwxrwxr-x 4 bwillcox bwillcox   4096 Dec 28 16:37 objects
-rw-rw-r-- 1 bwillcox bwillcox    966 Dec 28 16:37 packed-refs
drwxrwxr-x 5 bwillcox bwillcox   4096 Dec 28 16:37 refs
ubuntu % pwd
/home/bwillcox/go/src/github.com/infonova/prometheusbeat/prometheusbeat

--- a/README.md
+++ b/README.md
@@ -148,8 +148,8 @@ make clean
 To clone Prometheusbeat from the git repository, run the following commands:
 
 ```
-mkdir -p ${GOPATH}/github.com/infonova/prometheusbeat
-cd ${GOPATH}/github.com/infonova/prometheusbeat
+mkdir -p ${GOPATH}/github.com/infonova
+cd ${GOPATH}/github.com/infonova
 git clone https://github.com/infonova/prometheusbeat
 ```
 

